src/core/grpc_service.proto

```diff
@@ -264,26 +264,18 @@ message SharedMemoryControlRequest
     //@@
     message CUDASharedMemoryIdentifier
     {
-      //@@  .. cpp:var:: string shared_memory_key
-      //@@
-      //@@     The name of the system shared memory region that holds the
-      //@@     cudaIPC handle.
-      //@@
-      string shared_memory_key = 1;
-
-      //@@  .. cpp:var:: uint64 offset
+      //@@  .. cpp:var:: string name
       //@@
-      //@@     The offset of the cudaIPC handle from the start of the shared
-      //@@     memory region.
-      //@@     start = offset, end = offset + size;
+      //@@     The name of the system shared memory region registered in TRTIS
+      //@@     that holds the cudaIPC handle.
       //@@
-      uint64 offset = 2;
+      string name = 1;
```

Comment:
"name" is confusing since we also use "name" for the region being registered. Use "handle_region_name".